test_pgeocode.py

```diff
@@ -83,6 +83,27 @@ def test_download_dataset(temp_dir):
     assert len(res2.place_name.split(',')) > 1
 
 
+def test_download_gb_full_dataset(temp_dir):
+    assert not os.path.exists(os.path.join(temp_dir, 'GB_full.txt'))
+    nomi = Nominatim('gb_full')
+    # the data file was downloaded
+    assert os.path.exists(os.path.join(temp_dir, 'GB_full.txt'))
+    res = nomi.query_postal_code('BS6 5JR')
+
+    nomi2 = Nominatim('gb_full')
+    res2 = nomi.query_postal_code('BS65JR')
+
+    assert_array_equal(nomi._data.columns,
+                       nomi2._data.columns)
+    assert_array_equal(nomi._data_frame.columns,
+                       nomi2._data_frame.columns)
+    assert nomi._data.shape == nomi._data.shape
+    assert nomi._data_frame.shape == nomi._data_frame.shape
+
+    assert res.place_name == 'Bristol'
+    assert res2.place_name == 'Bristol'
+
+
```

Comment:
Could you please add a few tests for the `_Country` class?

Reply:
Of course, sorry for this!